src/net/sourceforge/kolmafia/RequestEditorKit.java

```diff
@@ -334,6 +323,7 @@ protected static final void applyPageAdjustments(
     } else if (location.startsWith("bigisland.php")) {
       IslandDecorator.decorateBigIsland(location, buffer);
     } else if (location.startsWith("casino.php")) {
+      // Should this be Lucky! instead?
```

Comment:
Yeah, it's confirmed the big rock is still from the casino and requires Lucky! now.